tests/tests_integration/test_api/test_templates.py

```diff
@@ -77,6 +79,33 @@ def new_template_instance(new_template_group_version):
     API_INSTANCES.delete(ext_id, new_version.version, instance.external_id)
 
 
+@pytest.fixture
+def new_view(new_template_group_version):
+    events = []
+    for i in range(0, 1001):
+        events.append(Event(external_id="test_evt_templates_" + str(i), type="test_templates", start_time=i * 1000))
```

Comment:
this will have potential race conditions when testing simultaneously with diff clients, better to gen a UUID or something.

Reply:
It doesn't fail if multiple clients are sending at the same time (or if it already exists). So it kind of works like insert or ignore at the moment, since I'm not cleaning them up afterwards. However, I can change it to generate it with UUIDs and then delete them afterwards instead, if that's current practice here.

Reply:
yes, that would be nice. we do like that generally.